### DISCUSSION TITLE
HELP - How do i change global shell glyph colors on 1.8 build 100?

### DISCUSSION THREAD

**Discussion by hiranokite** [moudey/Shell#131] [Q&A]
Im porting my old config to the new release, but i can't figure out how to set the glyph icons colors.
on the older 1.8 build, the syntax used was this
`glyph=[color.??, color.??]`
but on the new buid, if i save this line inside `theme { }` the shell won't load anymore.

**Comment by moudey** [comment]
```
shell
{
    set
    {
        theme.image
        {
	    color = [#f00, #0f0]
	    // or
	    color.color1 = #f00
	    color.color2 = #0f0
        }
    }
}
```
Please refer to this [doc](https://nilesoft.org/docs/syntax/set)

**Reply by hiranokite** [reply]
Thank you, worked perfectly. I'm now using the document to finish all the adjustments.
